eng/scripts/live-test-resource-cleanup.ps1

```diff
@@ -102,15 +119,31 @@ function HasValidOwnerTag() {
     Write-Warning " Resource group '$($ResourceGroup.ResourceGroupName)' has invalid owner tags: $($invalidOwners -join ',')"
   }
   if ($hasValidOwner) {
-      Write-Host " Skipping tagged exception resource group '$($ResourceGroup.ResourceGroupName)' with owners '$owners'"
-      return $true
+    Write-Host " Skipping tagged resource group '$($ResourceGroup.ResourceGroupName)' with owners '$owners'"
+    return $true
   }
   return $false
 }
 
+function HasValidAliasInName([object]$ResourceGroup) {
+    # check compliance (formatting first, then validate alias) and skip if compliant
+    if ($ResourceGroup.ResourceGroupName `
+      -match "^(rg-)?((t-|a-|v-)?[a-z,A-Z]{3,15})([-_]{1}.*)?$" `
```

Comment:
Tip: In general, I recommend using single quotes for PowerShell unless you intend to suppose expansion; however, I *greatly* recommend using single quotes for regex because it's common to use expansion operators like `$`. Frankly, I'm surprised this works and wonder if the trailing `$` is simply ignored since there's no variable name or scope operator immediately after it.

Reply:
That's a good catch. Will update.